test/parallel/test-intl.js

```diff
@@ -104,6 +104,12 @@ if (!common.hasIntl) {
     const numberFormat = new Intl.NumberFormat(['en']).format(12345.67890);
     assert.strictEqual(numberFormat, '12,345.679');
   }
+  // If list is specified and doesn't contain 'en-US' then return.
+  if (process.config.variables.icu_locales && !haveLocale('en-US')) {
+    common.printSkipMessage(
+      'detailed Intl tests because Node.js was built with small ICU.');
+    return;
```

Comment:
Why not `common.skip()` ?

Reply:
Not sure, I copied it from https://github.com/nodejs/node/blob/07e83f77faaa80f59743f44dc45bdf5e4f30aaaa/test/parallel/test-intl.js#L75-L81

Reply:
I believe `common.skip()` is for skipping everything in the file but `common.printSkipMessage()` can be used if you're just skipping a single test case in a file. But, uh, it's been a while since I've had to actually know how they worked.

Reply:
It's true, but the `return;` here effectively ends the test file.